L1Trigger/TrackerTFP/src/LayerEncoding.cc

```diff
@@ -0,0 +1,94 @@
+#include "L1Trigger/TrackerTFP/interface/LayerEncoding.h"
+
+#include <vector>
+#include <set>
+#include <algorithm>
+#include <cmath>
+
+using namespace std;
+using namespace trackerDTC;
+
+namespace trackerTFP {
+
+  LayerEncoding::LayerEncoding(const DataFormats* dataFormats) :
+    dataFormats_(dataFormats),
+    setup_(dataFormats->setup()),
```

Comment:
Add comment explaining what this constructor does. e.g. "Fill layerEncoding_ with all Tracker layers consistent with each (zT,tanL) digi bin in each eta sector".
P.S. Am I correct that most of the complexity here is to ensure that +- 1 digi bin is not erroneously excluded? e.g. If one didn't care about such precision, the loop over "boundaries" could be dropped? May be worth mentioning that too, so people don't spend ages trying to understand what is going on here.

Reply:
I don't understand your PS

Reply:
done